<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>receiptsCheckingWidget</class>
 <widget class="QWidget" name="receiptsCheckingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>378</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pointer les recettes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="2">
      <widget class="QPushButton" name="recordButton">
       <property name="text">
        <string>En&amp;registre</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLabel" name="labelAlpha">
       <property name="text">
        <string>Alpha</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QDateEdit" name="dateEndEdit"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="endDateLabel">
       <property name="text">
        <string>Date fin</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="usersLabel">
       <property name="text">
        <string>Utilisateurs</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>&amp;Quitter</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QComboBox" name="comboBoxUsers"/>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="revertButton">
       <property name="text">
        <string>A&amp;nnule</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="dateBeginEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="beginDateLabel">
       <property name="text">
        <string>Date début</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" rowspan="2" colspan="2">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>181</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="detailLabel">
       <property name="text">
        <string>Détail du règlement</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="detailEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
